From 5e55eb207247ebe2ba4674820bdca7377af37b8f Mon Sep 17 00:00:00 2001
From: Paul Durrant <paul.durrant@citrix.com>
Date: Mon, 1 Aug 2016 16:27:24 +0100
Subject: [PATCH 1/2] Add write to data updated to wake XAPI

Signed-off-by: Paul Durrant <paul.durrant@citrix.com>
---
 src/xenagent/service.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/xenagent/service.cpp b/src/xenagent/service.cpp
index 170c2dd..e29ecd9 100644
--- a/src/xenagent/service.cpp
+++ b/src/xenagent/service.cpp
@@ -231,6 +231,9 @@ void CXenAgent::StartShutdownWatch()
     m_device->StoreWrite("control/feature-reboot", "1");
     m_device->StoreWrite("control/feature-s3", "1");
     m_device->StoreWrite("control/feature-s4", "1");
+
+    // notify older-XAPI that features have been changed
+    m_device->StoreWrite("data/updated", "1");
 }
 
 void CXenAgent::StopShutdownWatch()
@@ -243,6 +246,9 @@ void CXenAgent::StopShutdownWatch()
     m_device->StoreRemove("control/feature-s3");
     m_device->StoreRemove("control/feature-s4");
 
+    // notify older-XAPI that features have been changed
+    m_device->StoreWrite("data/updated", "1");
+
     m_device->StoreRemoveWatch(m_ctxt_shutdown);
     m_ctxt_shutdown = NULL;
 }
-- 
2.1.1

