From 6492606afc03a7277241f38d452c990c1cfd26d1 Mon Sep 17 00:00:00 2001
From: Paul Durrant <paul.durrant@citrix.com>
Date: Mon, 1 Aug 2016 17:34:50 +0100
Subject: [PATCH 2/2] Also check platform/timeoffset

Signed-off-by: Paul Durrant <paul.durrant@citrix.com>
---
 src/xenagent/service.cpp | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/xenagent/service.cpp b/src/xenagent/service.cpp
index e29ecd9..a1c4dc1 100644
--- a/src/xenagent/service.cpp
+++ b/src/xenagent/service.cpp
@@ -397,8 +397,10 @@ void CXenAgent::AdjustXenTimeToUTC(FILETIME* now)
         return;
 
     std::string offs;
-    if (!m_device->StoreRead(vm + "/rtc/timeoffset", offs))
-        return;
+    if (!m_device->StoreRead(vm + "/rtc/timeoffset", offs)) {
+        if (!m_device->StoreRead("platform/timeoffset", offs))
+            return;
+    }
 
     long offset = (long)atoi(offs.c_str());
 
-- 
2.1.1

